lib/commands/command-current.bash

```diff
@@ -48,7 +50,8 @@ current_command() {
 
   # printf "$terminal_format" "PLUGIN" "VERSION" "SET BY CONFIG" # disbale this until we release headings across the board
   if [ $# -eq 0 ]; then
-    for plugin in $(asdf plugin list); do
+    # shellcheck disable=SC2119
+    for plugin in $(plugin_list_command); do
```

Comment:
`asdf` is a binary on the user's path, whereas `plugin_list_command` is a Bash function already in memory. While the code looks similar invoking `asdf` results in a new process that re-executes all the code in `bin/asdf`.